build/win32/code.iss

```diff
@@ -20,9 +20,16 @@ Compression=lzma
 SolidCompression=yes
 AppMutex={code:GetAppMutex}
 SetupMutex={#AppMutex}setup
+// {{SQL CARBON EDIT}} Use separate icons for non-stable builds
+#if Quality == "stable"
 WizardImageFile="{#RepoDir}\resources\win32\inno-big-100.bmp,{#RepoDir}\resources\win32\inno-big-125.bmp,{#RepoDir}\resources\win32\inno-big-150.bmp,{#RepoDir}\resources\win32\inno-big-175.bmp,{#RepoDir}\resources\win32\inno-big-200.bmp,{#RepoDir}\resources\win32\inno-big-225.bmp,{#RepoDir}\resources\win32\inno-big-250.bmp"
 WizardSmallImageFile="{#RepoDir}\resources\win32\inno-small-100.bmp,{#RepoDir}\resources\win32\inno-small-125.bmp,{#RepoDir}\resources\win32\inno-small-150.bmp,{#RepoDir}\resources\win32\inno-small-175.bmp,{#RepoDir}\resources\win32\inno-small-200.bmp,{#RepoDir}\resources\win32\inno-small-225.bmp,{#RepoDir}\resources\win32\inno-small-250.bmp"
 SetupIconFile={#RepoDir}\resources\win32\code.ico
+#else
```

Comment:
ideally we could avoid spreading this stable\insiders logic out of the build scripts.  Since this isn't aligned with the ultimate direction we'll be going with this type of customization.  Which is to use build-time configuration mixins in our distro branch.

Reply:
If it doesn't become possible I would recommend not taking this until we do our distro changes.

Reply:
I'll look into doing run-time replacement of the file instead. 

Do we have any sort of timeline for build-time configuration? I haven't heard anything about actually aligning with vscode on that.